voluncheer/templates/profiles/profile.html

```diff
@@ -287,27 +283,30 @@ <h2>PAST OPPORTUNITIES</h2>
         </div>
 
         <div class="sidenav-url">
+            {% if user.pk is curr_user.pk %}
             <div class="url">
                 <a href="{% url 'home' %}" class="active" >PROFILE</a>
                 <hr align="center">
             </div>
+            {% endif %}
             {% if user.pk is curr_user.pk %}
-
             <div class="url">
                 <a href="{% url 'saved_events' %}" >MY VOLUNCHEERS</a>
                 <hr align="center">
             </div>
             {% endif %}
             {% if user.pk is not curr_user.pk %}
-            <div class="url">
-                <a href="#" onclick="toggleChatVol()">CHAT</a>
-                <hr align="center">
-            </div>
+                <div class="url">
+                    <a href="#" onclick="toggleChatVol()">CHAT</a>
+                    <hr align="center">
+                </div>
             {% endif %}
+            {% if user.pk is curr_user.pk %}
```

Comment:
Create Post is for all users. Every user can post on others gallery. This also goes to viewing gallery

Reply:
Wait, but this just doesn't make sense to me.... How come others can determine the content in my gallery but now myself?